content-src/components/HighlightContext/HighlightContext.js

```diff
@@ -2,30 +2,42 @@ const React = require("react");
 const highlightTypes = require("./types");
 const Tooltip = require("components/Tooltip/Tooltip");
 const getRelativeTime = require("lib/getRelativeTime");
+const classNames = require("classnames");
 
 const HighlightContext = React.createClass({
+  getDefaultProps() {
+    return {placeholder: false};
+  },
   render() {
     let timestamp;
     const type = this.props.type;
     const {icon, label, showTimestamp, tooltip} = Object.assign({}, highlightTypes[type], this.props);
     if (this.props.date && showTimestamp !== false) {
       timestamp = getRelativeTime(this.props.date);
     }
-    return (<div className="highlight-context tooltip-container">
-      <span className="hc-icon">
-        <span ref="icon" className={`icon icon-${icon}`} /><span className="sr-only">{this.props.type}</span>
-      </span>
-      <span ref="label" className="hc-label">{label}</span>
-      <span hidden={!timestamp} ref="timestamp" className="hc-timestamp">{timestamp}</span>
-      {tooltip && <Tooltip ref="tooltip" label={tooltip} />}
-    </div>);
+    return (
+      <div className={classNames("highlight-context", "tooltip-container", {placeholder: this.props.placeholder})}>
+        <span className="hc-icon">
+          <span ref="icon" className={`icon icon-${icon}`} /><span className="sr-only">{this.props.type}</span>
```

Comment:
nit: `{placeholder: this.props.placeholder}` and `this.props.type` can just be `{placeholder}` and `type` if you change the `const type` above to `const {placeholder, type} = this.props`